pytest/test_all.py

```diff
@@ -75,3 +75,21 @@ def test_issue_433():
         """
     )
     assert out == 25
+
+def test_notebook():
+    import os
+    import subprocess
+    import sys
+    from pathlib import Path
+
+    result = subprocess.run(
+        [
+            sys.executable,
+            "-m",
+            "pytest",
+            "--nbval",
+            str(Path(__file__).parent / "test_nb.ipynb"),
```

Comment:
Not sure if coverage is propagated through recursive calls to `pytest`. If you would prefer we can just add a second `pytest` call to the CI but I slightly prefer calling it from within the existing tests so that a single `pytest` call tests everything.

Reply:
You could instead simply add the `--nbval` arg to the `pytest` command in `tests.yml` - then it will find tests in both notebooks and *.py files.

Reply:
Oh, I didn't even realize it would combine them. Cool!